istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -463,6 +466,8 @@ func (o *RemoteSecretOptions) addFlags(flagset *pflag.FlagSet) {
 		"Name of the local cluster whose credentials are stored "+
 			"in the secret. If a name is not specified the kube-system namespace's UUID of "+
 			"the local cluster will be used.")
+	flagset.StringVar(&o.ServerOverride, "server", "",
```

Comment:
Please use exactly the same short alternate and help text as _kubectl_: `-s, --server='': The address and port of the Kubernetes API server`

Reply:
Fixed, thanks!